tools/monorepo-utils/src/index.ts

```diff
@@ -9,12 +9,39 @@ import chalk from 'chalk';
  * Internal dependencies
  */
 import CodeFreeze from './code-freeze/commands';
+import Slack from './slack/commands/slack';
+import { Logger } from './core/logger';
+import { isGithubCI } from './core/environment';
 
-console.log(
-	chalk.rgb( 150, 88, 138 ).bold( figlet.textSync( 'WooCommerce Utilities' ) )
-);
+if ( ! isGithubCI() ) {
+	Logger.notice(
+		chalk
+			.rgb( 150, 88, 138 )
+			.bold( figlet.textSync( 'WooCommerce \n Utils' ) )
+	);
+}
 
-export const program = new Command()
+const program = new Command()
 	.name( 'utils' )
 	.description( 'Monorepo utilities' )
-	.addCommand( CodeFreeze );
+	.addCommand( CodeFreeze )
+	.addCommand( Slack );
+
+program.exitOverride();
+
+const run = async () => {
+	try {
+		//  parseAsync handles cases where the action is async and not async.
+		await program.parseAsync( process.argv );
```

Comment:
To properly handle async actions we should be calling parseAsync. if you have any async actions in your CLI you should run parseAsync because it handles both.